### DISCUSSION TITLE
Ratespiel fürs Museum

### DISCUSSION THREAD

**Discussion by Eskimue** [Wendelpunkt/ToypadChallenge#10] [Ideas]
Wir betreiben als Verein ein kleines Museum (Astronomiemuseum), in dem wir bereits einige Softwareanwendungen selbst programmieren (C#), um unseren Besuchern auch aktiv teilhaben zu lassen. Vielleicht gibt es die Möglichkeit, ein interaktives Spiel für unsere jüngeren Besucher zu entwickeln. Dieses Spiel könnte Aktivitäten wie das Anordnen von Objekten in der richtigen Reihenfolge oder kleine Ratespiele beinhalten. Zumindest ist es spannend, dass man haptisch Wissen vermitteln kann.

**Comment by tomwendel** [comment]
Die Idee gefällt mir gut. Im Museum hast du halt auch immer die Herausforderung, dass die beweglichen Teile dann nicht wieder verschwinden :D

**Reply by EdenFighter** [reply]
Zu dem verschwinden Problem: Da wäre es vermutlich gut oberhalb eine kleine Vorrichtung anzubringen an der die Figuren mit einem Drahtseil oder so festgemacht werden.

**Comment by EdenFighter** [comment]
Ich könnte mir richtig gut vorstellen wie man dann kleine Figuren der Planeten oder von Sonden, ISS, etc. hat, die man auf das Pad stellen kann. Man könnte auch sehr gut eine Zuordnung machen von Objekten auf die linke oder rechte Seite des Pads.
